blaise82/l.m.s-server
# #170868357 Set up node js

#### What does this PR do?
- set up express
- set up dotenv
- set up eslint

#### Description of Task to be completed?
- set up express
helps in fast-tracking development of server-based applications
- set up dotenv
loads environment variables from a .env file into process.env
- set up eslint
JavaScript linting utility

#### How should this be manually tested?
- clone this repository
- check out to _ch-setup-node-express-170868357  branch
- run `npm run install`
- run `npm start`
- open localhost:8080 in your browser 

#### What are the relevant pivotal tracker stories?
#170868357

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-01-23 at 16 17 34" src="https://user-images.githubusercontent.com/52036624/72992168-039e9980-3dfc-11ea-8c2f-8236f56ca3aa.png">
